test/common/testobjects.go

```diff
@@ -88,18 +87,18 @@ func NewTestTargetObject(typeConfig typeconfig.Interface, namespace string, fixt
 	}
 
 	federate.SetBasicMetaFields(obj, typeConfig.GetTarget(), "", namespace, "test-e2e-")
```

Comment:
Why does `SetBasicMetaFields` accept a generated name if it is a test-only field?

Reply:
The intention was to use the same function to set fields including test code. If that's not desirable, I can update a separate function for test.